.vscode/settings.json

```diff
@@ -3,5 +3,12 @@
 	"editor.codeActionsOnSave": {
```

Comment:
In general we don't usually want to commit anything that is in our .vscode directory as that should remain local. Can we please add this line to the `.gitignore` file?
```
.vscode/
```

Reply:
Hey Liam, I just pushed .vscode/ to the Gitinore file :)